metadata-ingestion/src/datahub/ingestion/source/looker/looker_config.py

```diff
@@ -202,7 +202,6 @@ def external_url_defaults_to_api_config_base_url(
     def stateful_ingestion_should_be_enabled(
         cls, v: Optional[bool], *, values: Dict[str, Any], **kwargs: Dict[str, Any]
     ) -> Optional[bool]:
-
```

Comment:
would it be possible to revert some of these whitespace changes?

I definitely do want to upgrade our `black` version, but might be nicer to do it in a separate PR

We're currently pinned to 22.12.0 (https://github.com/datahub-project/datahub/blob/4250feb6a518c529d8fb3a4a93205416ab296b73/metadata-ingestion/setup.py#L426)